tests/system/python/packages/test_multiple_assets.py

```diff
@@ -143,10 +149,15 @@ def verify_ping(fledge_url, skip_verify_north_interface, wait_time, retries):
     return ping_result
 
 
-def verify_asset(fledge_url, total_assets):
-    get_url = "/fledge/asset"
-    result = utils.get_request(fledge_url, get_url)
-    assert len(result), "No asset found"
+def verify_asset(fledge_url, total_assets, count, wait_time):
+    for i in range(count):
+        get_url = "/fledge/asset"
+        result = utils.get_request(fledge_url, get_url)
+        asset_created = len(result)
+        if (total_assets == asset_created):
+            print("Total {} asset created".format(asset_created))
+            return
+        time.sleep(wait_time * 6)
```

Comment:
why *6 is required?

Reply:
Fledge takes 60 seconds to create 100 assets, Here wait_time is 10.
I added sleep for "wait_time * 6", So that we can changes value from the jenkins job in future if required.